actionpack/test/dispatch/static_test.rb

```diff
@@ -199,7 +219,7 @@ class StaticTest < ActiveSupport::TestCase
 
   def setup
     @root = "#{FIXTURE_LOAD_PATH}/public"
-    @app = ActionDispatch::Static.new(DummyApp, @root, "public, max-age=60")
+    @app = ActionDispatch::Static.new(DummyApp, @root, "public, max-age=60", ACCESS_CONTROL_HEADERS)
```

Comment:
These headers are not needed for all tests. Would instantiate `@app = …` in the specific test case instead.